python/cuml/test/test_fil.py

```diff
@@ -376,6 +376,26 @@ def test_output_storage_type(storage_type, small_classifier_and_preds):
     assert np.allclose(fil_preds, xgb_preds_int, 1e-3)
 
 
+@pytest.mark.skipif(has_xgboost() is False, reason="need to install xgboost")
+@pytest.mark.parametrize('storage_type',
+                         ['dense', 'sparse', 'sparse8'])
+@pytest.mark.parametrize('blocks_per_sm', [0, 1, 2, 3, 4])
```

Comment:
could probably afford to reduce this matrix - currently this will run 15 times..